test/Microsoft.OpenApi.Trimming.Tests/Microsoft.OpenApi.Trimming.Tests.csproj

```
@@ -0,0 +1,17 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net8.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+    <PublishTrimmed>true</PublishTrimmed>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
```

```suggestion
    <TrimmerSingleWarn>false</TrimmerSingleWarn>
    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
```

Adding `TrimmerSingleWarn=false` is nice because it will show all the warnings from referenced assemblies - and won't collapse them into a single warning for the whole assembly.

---

Good tip!